build/Dockerfile

```diff
@@ -102,11 +102,14 @@ RUN \
     fonts-sil-gentium \
     fonts-sil-gentium-basic &&\
     rm -f ./ttf-mscorefonts-installer_3.8_all.deb &&\
-    echo "deb https://httpredir.debian.org/debian/ buster-backports main contrib non-free" >> /etc/apt/sources.list &&\
+    # See https://github.com/nextcloud/docker/issues/380.
+    mkdir -p /usr/share/man/man1mkdir -p /usr/share/man/man1 &&\
+    curl -L "https://download.documentfoundation.org/libreoffice/stable/7.2.0/deb/x86_64/LibreOffice_7.2.0_Linux_x86-64_deb.tar.gz" | tar -xz &&\
```

Comment:
It will certainly work on `amd64` architectures, but for not for `arch64` ones.

Note the `x86_64` part of the URL 😄